storops/unity/resource/alert_config_snmp.py

```diff
@@ -0,0 +1,68 @@
+# coding=utf-8
+# Copyright (c) 2015 EMC Corporation.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from __future__ import unicode_literals
+
+import logging
+
+from storops.unity.resource import UnityResource, UnityResourceList
+
+LOG = logging.getLogger(__name__)
+
+
+class UnityAlertConfigSNMPTarget(UnityResource):
+
+    @classmethod
+    def create(cls, cli, target_address, auth_protocol=None, username=None,
+               auth_password=None, priv_protocol=None, priv_password=None,
+               snmp_version=None, community=None):
+        req_body = cli.make_body(targetAddress=target_address,
```

Comment:
`targetAddress` should be `address`

Reply:
I think this should be followed with create operation~ 
Use 'address' will gave 'wrong parameter ...' error